src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -3591,24 +3546,35 @@ public final class Warnings implements Serializable {
 
 
     public Warnings build() {
-      return new Warnings(
+      String missing = "";
```

Comment:
It's similar to AutoValue's build() method, which enforces non-nullness on required fields. If a required field is missing, then the name of that field is appended to the String missing.

As of this comment, changed so that
- that line only shows up if there are required fields
- throw an exception if any required fields are null, with variable [missing] as the error message.